frontend/awx/views/schedules/hooks/getScheduleResourceUrl.tsx

```diff
@@ -0,0 +1,39 @@
+import { RouteObj } from '../../../../Routes';
+import { Schedule } from '../../../interfaces/Schedule';
+
+export function getScheduleResourceUrl(schedule: Schedule): string {
+  const getResourceTypeSubString: (type: string) => string = (type) => {
+    let urlSubString = '';
+    switch (type) {
+      case 'job':
```

Comment:
you can just return the string inside each of these `case` statements since you don't do anything else with the string below